dist/api_data.py

```diff
@@ -1342,6 +1342,9 @@ def __ge__(self, other):
 'WT_SESSION.close' : Method([]),
 
 'WT_SESSION.compact' : Method([
+    Config('free_space_target', '20MB', r'''
```

Comment:
Seems that we should consider 0 as a specific value or set the min to 1. What do you all think?